api/upload/uploadRouter.js

```diff
@@ -9,6 +9,7 @@ const router = express.Router();
 const Upload = require('./uploadModel');
 require('dotenv').config();
 const authRequired = require('../middleware/authRequired');
+const { date } = require('faker');
```

Comment:
Why do we need to import `date` from `faker`?

Reply:
removed it